docs/source/README.md

```diff
@@ -5,7 +5,7 @@
 
 ![OmicLearn Tests](https://github.com/MannLabs/OmicLearn/workflows/OmicLearn%20Tests/badge.svg)
```

Comment:
On line 1, could you please update the link to OmicLearn logo as https://raw.githubusercontent.com/MannLabs/OmicLearn/master/omiclearn/utils/omic_learn_black.png since it is broken on https://omiclearn.readthedocs.io/?